torchvision/datasets/kitti.py

```diff
@@ -0,0 +1,184 @@
+import os
+from collections import namedtuple
+from typing import Any, Callable, NamedTuple, Optional, Tuple
+from urllib.error import URLError
+
+import pandas as pd
+from PIL import Image
+
+from .utils import download_and_extract_archive
+from .vision import VisionDataset
+
+
+class Kitti(VisionDataset):
+    """`KITTI <http://www.cvlibs.net/datasets/kitti>`_ Dataset.
```

Comment:
We should also mention the pandas dependency here in the docstring

Reply:
Removed pandas dependency.